bitcoin/src/bip32.rs

```diff
@@ -313,6 +313,7 @@ impl<'a> From<&'a [ChildNumber]> for DerivationPath {
     fn from(numbers: &'a [ChildNumber]) -> Self { DerivationPath(numbers.to_vec()) }
 }
 
+
```

Comment:
You've left an empty line here but I don't care.

Reply:
Removed, I also dislike unrelated changes in diffs :)